plugins/linux-capture/pipewire.c

```diff
@@ -439,17 +439,21 @@ static uint32_t build_format_params(obs_pipewire_data *obs_pw,
 	params =
 		bzalloc(2 * obs_pw->format_info.num * sizeof(struct spa_pod *));
 
+	if (!check_pw_version(&obs_pw->server_version, 0, 3, 33))
```

Comment:
Hm, shouldn't it be checking 0.3.24 here?

Reply:
No, since we are using the `FLAT_DONT_FIXATE` which was introduced in PipeWire 0.3.33.